### PR TITLE
Weather app pull request 

### PR DESCRIPTION
**Pull request by Grifo89**
# This pull request has the following feature:
- A completely functional weather app.

### CODE REVIEW

File: .eslintrc.json
Comment (shloch) [inline .eslintrc.json]: Please use same config provided by Microverse for all js projects and re-run linter error checks [ [config here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) ]

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @Grifo89 

Nice job so far, this is my first visual observation after testing app on browser below :
- firstly i cloned, entered right folder , switched to appropriate branch and ran weback (so you be sure i followed the right steps)
` git clone https://github.com/Grifo89/Weather-app-javascript.git; cd Weather-a*; git checkout fetching; npm i; npm run-script build`
- then when i open app, no matter what city i enter, i get this
Screen Shot 2020-03-06 at 9 33 55 AM

----> remove the `console.log(err);` from your `fetch.js` file (line 8)
You can instead display somewhere on the screen so that usuer can see the error

### lets get to debugging, waiting for another review (from viewing your code, it's clear you're on the right track....great job)

**Comment by Grifo89** [comment]
Hello, @shloch actually is not working because you need to create a .env file in the root directory with your own API as is specified on the README file.
image

**Comment by shloch** [comment]
Alright..thanks for updates, sorry I never saw that after filling for next review...can you please include your key if it's nothing discrete (for example, if you created it just for dev sake of this project ?) Thanks

**Comment by Grifo89** [comment]
I added my api_key to the project. It must work right away now.

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @Grifo89 

Thanks for the updates, it works......great job 
However, we got 02 more updates to make which am sure you can do to finish with the requirements

- there should a toggle functionality such that we can switch from Celcius to Fahrenheit and vice verse [ [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) ]

- App should  change its appearance based on the received data (i believe cold whether some snow, hot some sand ...you get the idea). You can do this for Background image [ [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) ]

### thats all am expecting from you to get a thumbs up bro.......
HAPPY CODING

**Comment by Grifo89** [comment]
Thanks for the review. Although regarding the appearance, the animated icon changes depending on the weather and time of the day. So I think what it's missing is the toggle button.

**Review by bellom — APPROVED** [review]
## Hi @Grifo89  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**
